Gemfile

```diff
@@ -20,11 +24,10 @@ gem "jekyll", "3.6.3"
 
 # If you have any plugins, put them here!
 group :jekyll_plugins do
-   gem "jekyll-feed", "~> 0.6"
+   gem "jekyll-feed", "~> 0.16"
    gem 'jekyll-paginate', '~> 1.1'
    gem 'jekyll-seo-tag'
    gem 'jekyll-gist'
-   gem 'jekyll-livereload'
```

Comment:
This plugin is deprecated. The functionality was added to jekyll with the `--livereload` flag in v3.7.0. https://github.com/RobertDeRose/jekyll-livereload#deprecation-warning